lesson4/_homework/task_01/script.js

```diff
@@ -0,0 +1,22 @@
+
+//                    Adapt Academy 2019 lesson 04 - JS | task 01 by Donatas Dereškevičius                     
+
+const countryList = [ 
+    {"name": "Afghanistan", "code": "AF"}, 
+    {"name": "Åland Islands", "code": "AX"}, 
+    {"name": "Albania", "code": "AL"}, 
+    {"name": "Algeria", "code": "DZ"}
+    ]
+
+    const getCountryByCode = (data) => {
+
+        input = document.querySelector(".form-input").value;
```

Comment:
I can see you're taking data from the existing input field rather than a function parameter. Maybe you should try creating an event listener for a button which executes getCountryByCode() providing entered data to that input field?